internal/build/container_ops.go

```diff
@@ -97,10 +97,9 @@ func copyDirWindows(ctx context.Context, ctrClient client.CommonAPIClient, conta
 		&dcontainer.Config{
 			Image: info.Image,
 			Cmd: []string{
-				"xcopy",
-				fmt.Sprintf(`c:\windows\%s`, baseName),
-				dst,
-				"/e", "/h", "/y", "/c", "/b",
+				"cmd",
+				"/c",
+				fmt.Sprintf(`echo %s|xcopy /e /h /y /c /b c:\windows\%s %s`, fileOrDir, dst[3:], dst),
```

Comment:
Would you mind clarifying what the flags are for me?

Reply:
Is there a way to make `xcopy /i` work here instead of piping in the answer?

> /i  -- If Source is a directory or contains wildcards and Destination does not exist, xcopy assumes Destination specifies a directory name and creates a new directory. Then, xcopy copies all specified files into the new directory. By default, xcopy prompts you to specify whether Destination is a file or a directory.

https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/xcopy#:~:text=If%20Source%20is%20a%20directory

Reply:
We tried `/i` and it didn't work for us for some reason. Piping seemed to be the suggestion based on some research. Can try again though. Is there a reason why one is better, other than a simpler command?

Reply:
Yeah, if I change to `/i` on your branch, it fails acceptance tests for me when I try it, though not the `container_opts_test` nor other units. I feel like that's an issue, we probably need some better test coverage, if only to serve as better documentation for our `xcopy` stuff. I'm worried that since this is such a hotspot for arbitrary input, we should probably bolster the coverage a bit.

Additionally, it might make it easier to iterate on this implementation in the future as well.

Reply:
Oh, nvm, there's a `SkipIf` in `container_ops_test`. Let's take that out as part of this work.

Reply:
I started a branch tweaking the tests as bit. It's stuck at what may be a bug in this implementation, but either way, adding some coverage to `container_ops_test` this PR shouldn't be too hard: 

https://github.com/micahyoung/buildpacks-pack/commit/a4182c87553c4eb671c5e2c3d9102254a320b1c1

Reply:
After trying a few different alternatives, it appears the current implementation is the best one. I tried plain `copy` for individual files but it did not preserve original ownership info, I also tried `tar -c | tar -x` but it doesn't handle symlinks properly.

Reply:
I added comments for the flags to a follow-up commit.